extra-credit/kittenbot.js

```diff
@@ -0,0 +1,216 @@
+/* *****************************************************************************
+Copyright 2020 Google Inc. All Rights Reserved.
```

Comment:
Nit: it's "Google LLC" and no "All Rights Reserved"

Reply:
Thanks - updated accross the files. Also found some that still said 2016 :)